From: YueHaibing <yuehaibing@huawei.com>
Date: Wed, 17 Apr 2019 23:05:54 +0800
Subject: [PATCH] scsi: lpfc: Make lpfc_sli4_oas_verify static
References: bsc#1136217,jsc#SLE-4722
Git-commit: c70929752511cb73dfb940d4c3f00e75ab55ef67
Patch-mainline: v5.2-rc1

Fix sparse warning:

drivers/scsi/lpfc/lpfc_init.c:13091:1: warning:
 symbol 'lpfc_sli4_oas_verify' was not declared. Should it be static?

Signed-off-by: YueHaibing <yuehaibing@huawei.com>
Acked-by: James Smart <james.smart@broadcom.com>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 drivers/scsi/lpfc/lpfc_init.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/scsi/lpfc/lpfc_init.c b/drivers/scsi/lpfc/lpfc_init.c
index f29115abb9f6..bc4dcf1a19fc 100644
--- a/drivers/scsi/lpfc/lpfc_init.c
+++ b/drivers/scsi/lpfc/lpfc_init.c
@@ -13101,7 +13101,7 @@ lpfc_io_resume(struct pci_dev *pdev)
  * is destroyed.
  *
  **/
-void
+static void
 lpfc_sli4_oas_verify(struct lpfc_hba *phba)
 {
 
-- 
2.16.4

